client/src/components/scans/scanListItem.js

```diff
@@ -205,6 +205,20 @@ class ScanListItem extends React.Component {
   renderActions() {
     const { item, onSummaryDownload, onDetailedDownload, onPause, onCancel, onStart, onResume } = this.props;
 
+    let downloadActions = null;
+    if (_.get(item, 'most_recent.report_id')) {
+      downloadActions = (
+        <DropdownButton key="downLoadButton" title="Download" pullRight id={`downloadButton_${item.id}`}>
+          <MenuItem eventKey="1" onClick={() => onSummaryDownload(_.get(item, 'most_recent.report_id'))}>
+            Summary Report
+          </MenuItem>
+          <MenuItem eventKey="2" onClick={() => onDetailedDownload(_.get(item, 'most_recent.report_id'))}>
+            Detailed Report
+          </MenuItem>
+        </DropdownButton>
+      );
+    }
```

Comment:
Iterate: Like centralizing it works well. Doing this points out we've got some pretty heavy HTML repetition going on, something to take a look at as clean up maybe